data-sources/azure-infrastructure-monitoring/config.yml

```diff
@@ -0,0 +1,8 @@
+id: azure-infrastructure-monitoring
+displayName: Azure Infrastructure monitoring
+description: |
+  Azure Infrastructure monitoring allows monitoring of various Azure services.
+install:
+  primary:
+    link:
+      url: https://docs.newrelic.com/docs/infrastructure/microsoft-azure-integrations/get-started/activate-azure-integrations/
```

Comment:
Lets use the `microsoft-azure` core data source instead. The UI provides a link to the same doc if needed